test/index.js

```diff
@@ -614,6 +689,21 @@ describe('simple-keyring', function () {
     });
   });
 
+  describe('exportAccount', function () {
+    it('should return a hex-encoded private key', async function () {
+      const { address } = testAccount;
+      const simpleKeyring = new SimpleKeyring([testAccount.key]);
+      const privKeyHexValue = await simpleKeyring.exportAccount(address);
+      expect(`0x${privKeyHexValue}`).toBe(testAccount.key);
```

Comment:
`0x${privKeyHexValue}` is the expected value, yes? If so I think it'd make more sense to flip this around.

Reply:
fixed